hasher-matcher-actioner/hmalib/fetching/bank_store.py

```diff
@@ -303,7 +304,7 @@ def _handle_delete(self, key: TUpdateRecordKey):
             self._import_bank.bank_id, str_key
         )
         if member:
-            self.banks_table.remove_bank_member(member.bank_member_id)
+            bank_ops.remove_bank_member(self.banks_table, member.bank_member_id)
```

Comment:
Use bank_ops.remove_bank_member which does the coordination of removing signals and the member itself.